src/navigator/TopBarButton.tsx

```diff
@@ -13,17 +13,28 @@ interface CommonProps {
   testID?: string
   onPress: () => void
   eventName?: AnalyticsEventType
+  eventProperties?: any
```

Comment:
We should type this fully to avoid errors as the code evolves.
Similar to https://github.com/valora-inc/wallet/blob/2b28b691640c52bdeeb1b49a7751c5bdf3b764fe/src/analytics/ValoraAnalytics.ts#L118-L121
I'm happy to help.